src/main/java/org/takes/facets/fallback/FbLog4j.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.takes.facets.fallback;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+import lombok.EqualsAndHashCode;
+import org.apache.log4j.Logger;
+import org.takes.Response;
+import org.takes.misc.Opt;
+import org.takes.rq.RqHref;
+import org.takes.rq.RqMethod;
+
+/**
+ * Fallback that logs all problems through Log4J.
+ *
+ *
+ * @author Igor Piddubnyi (igor.piddubnyi@gmail.com)
+ * @version $Id$
+ * @since 0.25
+ */
+@EqualsAndHashCode(callSuper = true)
+public final class FbLog4j extends FbWrap {
+
+    /**
+     * Log4J logger.
+     */
+    private static final Logger LOGGER = Logger.getLogger(FbLog4j.class);
+
+    /**
+     * Ctor.
+     */
+    public FbLog4j() {
+        super(
+            new Fallback() {
+                @Override
+                public Opt<Response> route(final RqFallback req)
+                    throws IOException {
+                    FbLog4j.log(req);
+                    return new Opt.Empty<Response>();
+                }
+            });
```

Comment:
@piddubnyi here the ending `);` should go to the next line, aligned with `super(`